pyspecProcScripts/lorentzian_to_gaussian.py

```diff
@@ -33,20 +39,24 @@ def L2G(s, axis='t2',
         fl.plot(temp,
                 alpha=0.6,
                 label='before')
-        fl.next('Lorentzian to Gaussian diagnostic -- frequency domain')
-        fl.plot(temp.C.ft('t2'),
-                alpha=0.6,
-                label='before')
     temp *= np.exp(-temp.fromaxis(axis)**2/2/(filter_width*pi/2)**2)
     temp /= np.exp(-abs(temp.fromaxis(axis))/filter_width)
     if fl is not None:
         fl.next('Lorentzian to Gaussian diagnostic -- time domain')
         fl.plot(temp,
                 alpha=0.6,
                 label='after')
+    temp.ft(axis)
+    if fl is not None:
         fl.next('Lorentzian to Gaussian diagnostic -- frequency domain')
-        fl.plot(temp.C.ft('t2'),
+        fl.plot(temp,
                 alpha=0.6,
                 label='after')
         axhline(y=0,color='k')
+    # {{{ actually apply the transform to the data and return
+    temp = s.C.ift(axis)
+    temp *= np.exp(-temp.fromaxis(axis)**2/2/(filter_width*pi/2)**2)
+    temp /= np.exp(-abs(temp.fromaxis(axis))/filter_width)
+    temp.ft(axis)
+    # }}}
```

Comment:
previously, the summed 1D data was returned, rather than the 2D data w/ transform applied/